src/main/java/com/ubc/cpsc304/hotelify/controller/ReviewController.java

```diff
@@ -0,0 +1,52 @@
+package com.ubc.cpsc304.hotelify.controller;
+
+
+import com.ubc.cpsc304.hotelify.controller.dto.ReviewRequestDto;
+import com.ubc.cpsc304.hotelify.controller.dto.ReviewResponseDto;
+import com.ubc.cpsc304.hotelify.entity.Review;
+import com.ubc.cpsc304.hotelify.service.ReviewService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Objects;
```

Comment:
use the code style .xml file that I sent to @Scottjrw , then bind it to a hotkey so that it does the code formatting for you